rfd/0133-connect-my-computer.md

```diff
@@ -482,7 +482,7 @@ The feature is supported only on macOS and Linux because the Teleport SSH agent
 Windows.
 
 Additionally, CMC is meant to be available only for Teleport Team plan users. To achieve this, the
-app fetches the `WebConfig` of the cluster and checks if the [`isUsageBasedBilling`](https://github.com/gravitational/teleport/blob/53b0118cff73fd5e9788a397d3b8a0aadc75d63a/web/packages/teleport/src/config.ts#L43-L44)
+app fetches the `WebConfig` of the cluster and checks if the [`isTeam`](https://github.com/gravitational/teleport/blob/a469c4ce2ba01eab296ef1fd9d141fb007227a42/web/packages/teleport/src/config.ts#L57)
```

Comment:
cc @ravicious @gzdunek this probably no longer applies, but i edited anyways

Reply:
I'm not sure if this needs to be edited, probably we should just remove it, along with the other mentions of the Team plan in the RFD (but this is out of scope for your work).
I can do this, so you can revert your changes.